src/main/java/com/contentful/java/cma/ModuleOrganizations.java

```diff
@@ -32,7 +32,7 @@
   final Async async;
 
   public ModuleOrganizations(Retrofit retrofit, Executor callbackExecutor) {
-    super(retrofit, callbackExecutor);
+    super(retrofit, callbackExecutor, null, null);
```

Comment:
Since the organisation is one level above spaces and environments, I would not expect the configuration to be interfering with the organisation queries.